src/server/controllers/todos.js

```diff
@@ -24,6 +24,7 @@ exports.update = (request, response) => {
   Todo.findById(request.param('id')).then((todo) => {
     if (!todo) return response.status(404);
     todo.title = request.body.title;
+    todo.completed = !todo.completed;
```

Comment:
Because the incoming request is a form, the value of "completed" would come in as a string (`"true"`) rather than a Boolean (`true`). As such, you would want to check the value of the "completed" field rather than just negating it.